tiled/media_type_registration.py

```diff
@@ -328,14 +328,8 @@ def close(self):
     ]:
         compression_registry.register(media_type, "lz4", LZ4Buffer)
 
-if modules_available("blosc"):
-    import blosc
-
-    # The choice of settings here is cribbed from distributed.protocol.compression.
```

Comment:
I cargo-culted these settings from `dask.distributed` in the first place without any benchmarking. They aren't directly transferrable to blsoc2, so I have reverted to just going with the defaults for now until we have time to benchmark.

Reply:
Do you want to retain this TODO as a comment or as a low-priority github issue?

Reply:
Good call. Done https://github.com/bluesky/tiled/issues/724